recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/menpo/condaci/raw/master/vs2008_patch.zip
+  sha256: c54143587f74a120f77987ea9a2d011e1c090709f07e3678f25aa1ffb93a6777
+
+build:
+  number: 0
+  skip: true  # [not win]
+  
+test:
+  commands:
+    - IF NOT EXIST %SCRIPTS%\setup_x64.bat exit 1
+    - IF NOT EXIST %SCRIPTS%\x64 exit 1
+
+about:
+  summary: Workaround to help CMake find the appropriate x64 compiler on VS 2008 Express with VS compiler for python on top.
```

Comment:
I guess `home` would be https://github.com/menpo/condaci or maybe the feedstock. Not sure.